tests/static/all_question_types.xml

```diff
@@ -85,6 +85,7 @@
 					</text>
 					<text id="question1-help">
 						<value>question1 en help</value>
+						<value form="markdown">question1 en help</value>
```

Comment:
Hmm, so we're duplicating the `<value>` content in `<value form="markdown">` as a flag to say treat this text as markdown. That design seems less than ideal. Is there a way for the phone to access the flag that says "treat all help text as markdown for this form"?